libnetwork/iptables/firewalld.go

```diff
@@ -159,13 +182,23 @@ func OnReloaded(callback func()) {
 }
 
 // Call some remote method to see whether the service is actually running.
-func checkRunning() bool {
+func checkRunning(ctx context.Context) (bool, error) {
```

Comment:
Should this now also be named "something something "ready""? As we're checking if it's ready, not "running"? (and the comment to also describe this?)